lib/components/GlobalHeader/GlobalSearchButton.tsx

```diff
@@ -0,0 +1,52 @@
+'use client';
+import { MagnifyingGlassIcon, XMarkIcon } from '@navikt/aksel-icons';
+import { DsButton, useRootContext } from '../';
+import { useIsDesktop } from '../../hooks/useIsDesktop';
+import styles from './globalSearch.module.css';
+
+export const GlobalSearchButton = ({
+  onClick,
+  expanded,
+}: {
+  onClick: () => void;
+  expanded: boolean;
+}) => {
+  const { languageCode } = useRootContext();
+  const { search, close } = getTexts(languageCode);
+  const isDesktop = useIsDesktop();
+  return (
+    <DsButton
```

Comment:
This will not scale properly on Safari without a height and a width and is a bug with DsButton. We have fixed this with ButtonIcon (that's why it works in GlobalMenu, but not here).